website/_posts/2023-03-31-what-is-testnet.md

```diff
@@ -0,0 +1,52 @@
+ 	
+**Testing 	new versions, patches and features on testnet.** 	it is vital for enhancing the security, load testing, Blockchain 	migration, integration tests, and etc. This is why you can now use 	the latest version of Pactus-GUI that works like clockwork and is 	extremely secure.
+ 	
+**Testing 	the functions of cryptocurrency without disrupting the mainnet.** In 	the testnet of Pactus Blockchain, you as a user or validator can use 	all the features to send, receive, bond or unbond your assets in the 	pactus-GUI.
+ 	
+**Development 	and simulation of smart contract features and DApps.** 	Note that many Blockchain projects like Pactus, can be used as 	smart-contract platforms. So the testnet can be designed in 	different stages to test and improve different applications of the 	network.
+
+Please note that after a certain period of time, when developers get satisfied with the functionality and debugging process, they launch Mainnet. Mainnet has its own independent genesis blocks; so the assets and addresses that users use in Testnet are not valid inside the new mainnet Blockchain anymore. In fact the whole Blockchain network starts working from the scratch and users and validators start to work with real tokens and transactions instead of the simulated ones. But since testnet has an important effect on the health and improvement of the network, in Pactus Blockchain, we have several advantages for the people who join the testnet.
+
+## Why should you join the testnet?
+There are different contributors in a Blockchain network;
+Some people are developers and programmers that use the Testnet to contribute, enhance and lead the project to a better future. Testnet is actually vital for them to perform their changes and try their innovations and ideas.
+
+But you might also be an end user; you might want to either be a validator and stake your assets to earn more Pactus coins, or you might want to learn **how to use the network for future** personal applications. Either way, you might find it handy to give it a try and be familiar with Pactus-GUI and its different features.
+
+**However, there is another advantage for end users who join the Testnet.** While using the network, running a node or testing the Pactus-GUI in Testnet period, you are contributing to the community to find improvement potentials, that’s why there are more advantages contemplated for the testnet users; Soon our mainnet will be launched and also Pactus network will be announced and listed in well-known exchanges. As an early contributor, you will have the advantage of getting some extra initial tokens at the beginning of mainnet launch. so that you can use it to run a validator node and earn more from staking your assets. This airdrop is considered for every individual who installs Pactus-GUI and starts a node in testnet.
```

Comment:
```suggestion
**However, there is another advantage for end users who join the Testnet.** While using the network, running a node or testing the Pactus-GUI in Testnet period, you are contributing to the community to find improvement potentials, that’s why there are more advantages contemplated for the Testnet users; Soon our Mainnet will be launched and also Pactus network will be announced and listed in well-known exchanges. As an early contributor, you will have the advantage of getting some extra initial tokens at the beginning of Mainnet launch. so that you can use it to run a validator node and earn more from staking your assets. This airdrop is considered for every individual who installs Pactus-GUI and starts a node in Testnet.
```